### PR TITLE
[bitnami/wavefront] Deprecate wavefront

### PR DESCRIPTION
**Pull request by corico44**
### Description of the change

Deprecate Wavefront.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ From time to time, one or more assets may be deprecated. In that situation, we w
 * Annotate components with the `to-be-deprecated: <date>` field in their config file setting the date when it will be removed. Add a deprecation notice in this `README.md` file as well.
 * Delete the config file and the associated `data/${name}` folder once the retention period has expired.
 
+### Deprecation notes
+
+### 2023-07-02
+
+Wavefront will no longer be supported.
```
Comment (carrodher) [inline README.md:42]: I would use a similar approach as we are using for other changelogs:
```suggestion
## Deprecation notes

### 2023-06
Wavefront
```

In this way we don't add an entry per day but per month.

Apart from that, could you please move this section to the end of the README? Just before the License

File: config/components/wavefront.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "wavefront",
-  "cpeProduct": "wavefront_by_vmware_nozzle"
+  "cpeProduct": "wavefront_by_vmware_nozzle",
+  "to-be-deprecated": "20230702"
```
Comment (carrodher) [inline config/components/wavefront.json:4]: We are not yet in July 🍹🏖️ 
```suggestion
  "to-be-deprecated": "20230602"
```

Comment (corico44) [reply config/components/wavefront.json:4]: The checklist says: Update JSON config in [vulndb](https://github.com/bitnami/vulndb) repo (`deprecated: <date>`) and a deprecation notice in `README.md`. Add the date in the format <yyyymmdd> (i.e. 20231231) with at least one month ahead.

File: README.md (hunk 1 above)
Comment (gongomgra) [reply README.md:42]: Agree with the list format and the monthly header.

File: README.md [hunk 3]
```diff
@@ -113,6 +113,11 @@ $ docker run bitnami/postgresql cat /opt/bitnami/postgresql/.spdx-postgresql.spd
 
 [Click here](https://github.com/bitnami/vulndb/issues/new/choose) to report a public vulnerability in the Bitnami ecosystem, or give us feedback about the project.
 
+## Deprecation notes
```
Comment (gongomgra) [inline README.md:116]: Can you also add a link to this section under the table of contents? Thanks!

**Review by gongomgra — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -113,6 +113,11 @@ $ docker run bitnami/postgresql cat /opt/bitnami/postgresql/.spdx-postgresql.spd
...
+
+### 2023-07
+Wavefront
```
Comment (gongomgra) [inline README.md:119]: Also convert this in a list, please:

```
* Wavefront
```

File: config/components/wavefront.json (hunk 2 above)
Comment (carrodher) [reply config/components/wavefront.json:4]: My fault. It's expected to set this date at least one month ahead.

**Review by carrodher — APPROVED** [review]

**Review by gongomgra — APPROVED** [review]
Thank you!

**Pull request merged**